tests/functional/ironicapi_controller_test.go

```diff
@@ -0,0 +1,202 @@
+/*
+Copyright 2022 Red Hat Inc.
```

Comment:
2023 :)

Reply:
oops, I rebased and fixed this nit in the last push.